### DISCUSSION TITLE
Change Postgres schema dinamically

### DISCUSSION THREAD

**Discussion by marcosjsfraga** [adonisjs/core#2015] [General]
I'm new in AdonisJS, and I need to develop an multi-tenant application based on PostgreSQL schemas. 
For this, i need to know how to change schema before connection in database. 
I think that this can be done using Middlewares, but i don know how.

Can someone help me?

**Comment by tavaresgerson** [comment]
Hi!

Exist this possibility: setting on schema in query https://docs.adonisjs.com/reference/database/query-builder#withschema
